src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Search/SearchDialogDto.cs

```diff
@@ -7,15 +7,33 @@ namespace Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.Dialogs.Que
 
 public sealed class SearchDialogDto : SearchDialogDtoBase
 {
+    /// <summary>
+    /// The content of the dialog in search results
+    /// </summary>
     [JsonPropertyOrder(100)] // ILU MAGNUS
```

Comment:
> // ILU MAGNUS

Memo to self - wat? 😂

Reply:
Ser ut som du har hjulpet @oskogstad ut av et dypt hull på et tidspunkt her :D Ikke helt åpenbart hva den attributten der er godt for, men jeg tør ikke røre den :D